<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Data on the parameters identified in the systematic review of articles related to Marburg virus disease. — marburg_parameter • epireview</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet"><link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet"><script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Data on the parameters identified in the systematic review of articles related to Marburg virus disease. — marburg_parameter"><meta name="description" content="This data set provides all extracted parameters for Marburg virus disease
(MVD)."><meta property="og:description" content="This data set provides all extracted parameters for Marburg virus disease
(MVD)."></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">epireview</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.4.2</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles"><li><a class="dropdown-item" href="../articles/epireview_overview.html">epireview</a></li>
    <li><a class="dropdown-item" href="../articles/load-filter-view-epidata.html">load-filter-view-epidata</a></li>
  </ul></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul><ul class="navbar-nav"><li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json"></form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/mrc-ide/epireview/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul></div>


  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Data on the parameters identified in the systematic review of articles related to Marburg virus disease.</h1>
      <small class="dont-index">Source: <a href="https://github.com/mrc-ide/epireview/blob/main/R/data.R" class="external-link"><code>R/data.R</code></a></small>
      <div class="d-none name"><code>marburg_parameter.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This data set provides all extracted parameters for Marburg virus disease
(MVD).</p>
    </div>


    <div class="section level2">
    <h2 id="format">Format<a class="anchor" aria-label="anchor" href="#format"></a></h2>
    <p>A csv file containing the following parameters:</p><ul><li><p>parameter_data_id = ID variable for the parameter.</p></li>
<li><p>article_id = ID variable for the article the outbreak data came from.</p></li>
<li><p>parameter_type = extracted parameter (see marburg_dropdown_parameters
for the full list of parameters extracted).</p></li>
<li><p>parameter_value = extracted parameter value as stated in the article.</p></li>
<li><p>parameter_unit = units of the extracted parameter (see
marburg_dropdown_parameters for the full list of parameter units).</p></li>
<li><p>parameter_lower_bound = minimum value of the parameter across any
dimension of disaggregation.</p></li>
<li><p>parameter_upper_bound = maximum value of the parameter across any
dimension of disaggregation.</p></li>
<li><p>parameter_value_type = whether the parameter value is the mean, median,
standard deviation.</p></li>
<li><p>parameter_uncertainty_single_value = extracted uncertainty if only a
single value provided.</p></li>
<li><p>parameter_uncertainty_single_type = uncertainty type if only a single
value is reported e.g. "Standard Error (SE)" (see
marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>parameter_uncertainty_lower_value = lower value of paired uncertainty.</p></li>
<li><p>parameter_uncertainty_upper_value = upper value of paired uncertainty.</p></li>
<li><p>parameter_uncertainty_type = uncertainty type if a pair of values are
provided, e.g. "CI90%", "CRI95%" (see marburg_dropdown_parameters
for the full list of options).</p></li>
<li><p>cfr_ifr_numerator = numerator of the cfr or ifr provided in the article.</p></li>
<li><p>cfr_ifr_denominator = denominator of the cfr or ifr provided in the
article.</p></li>
<li><p>distribution_type = if uncertainty is given as a distribution (see
marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>distribution_par1_value = value for distribution parameter 1.</p></li>
<li><p>distribution_par1_type = distribution parameter 1 type, e.g. shape, scale
(see marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>distribution_par1_uncertainty = whether the article reported uncertainty
for the distribution parameter estimates (TRUE/FALSE).</p></li>
<li><p>distribution_par2_value = value for distribution parameter 2.</p></li>
<li><p>distribution_par2_type = distribution parameter 2 type, e.g. shape, scale
(see marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>distribution_par2_uncertainty = whether the article reported uncertainty
for the distribution parameter estimates (TRUE/FALSE).</p></li>
<li><p>method_from_supplement = whether the parameter was taken from the
supplementary material of the article. TRUE/FALSE.</p></li>
<li><p>method_moment_value = time period, either "Pre outbreak", Start outbreak",
"Mid outbreak", "Post Outbreak", or "Other", if specified in the article.</p></li>
<li><p>cfr_ifr_method = whether the method used to calculate the cfr/ifr was
"Naive", "Adjusted", "Unknown".</p></li>
<li><p>method_r = method used to estimate the reproduction number (see
marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>method_disaggregated_by = how the parameter is disaggregated e.g. by Age,
Sex, Region.</p></li>
<li><p>method_disaggregated = whether the parameter is disaggregated (TRUE/FALSE).</p></li>
<li><p>method_disaggregated_only = whether only disaggregated data is available
(TRUE/FALSE).</p></li>
<li><p>riskfactor_outcome = the outcome(s) for which the risk factor was
evaluated e.g. "Death" or "Infection" (see marburg_dropdown_parameters
for the full list of options).</p></li>
<li><p>riskfactor_name = the potential risk factor(s) evaluated e.g. "Age" (see
marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>riskfactor_occupation = specific occupation(s) if occupation is a risk
factor (see marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>riskfactor_significant = either "Significant", "Not significant",
"Unspecified" or NA.</p></li>
<li><p>riskfactor_adjusted = either "Adjusted", "Not adjusted", "Unspecified".</p></li>
<li><p>population_sex = the sex of the study population, either "Female", "Male",
"Both", "Unspecified".</p></li>
<li><p>population_sample_type = how the study was conducted e.g. "Hospital based",
"Population based" (see 'Setting' in marburg_dropdown_parameters for the
full list of options).</p></li>
<li><p>population_group = population group e.g. "Healthcare workers" (see
marburg_dropdown_parameters for the full list of options).</p></li>
<li><p>population_age_min = minimum age in sample (if reported).</p></li>
<li><p>population_age_max = maximum age in the sample (if reported).</p></li>
<li><p>population_sample_size = total number of participants/samples tested.</p></li>
<li><p>population_country = country/countries that the study took place in.</p></li>
<li><p>population_location = location that the study took place e.g. region, city.</p></li>
<li><p>population_study_start_day = day study started (numeric - DD)</p></li>
<li><p>population_study_start_month = month study started (three letter
abbreviation e.g. "Feb")</p></li>
<li><p>population_study_start_year = year study started (numeric - YYYY)</p></li>
<li><p>population_study_end_day = day study ended (numeric - DD)</p></li>
<li><p>population_study_end_month = month study ended (three letter abbreviation
e.g. "Feb")</p></li>
<li><p>population_study_end_year = year study ended (numeric - YYYY)</p></li>
<li><p>genome_site = the portion of the pathogen’s genome used to estimate any
extracted parameters. I.e. the gene, gene segment, codon position, or a
more generic description (‘whole genome’ or ‘intergenic positions’).</p></li>
<li><p>genomic_sequence_available = whether the study sequenced new pathogen
isolates and their accession numbers have been provided for retrieval
from a public database. TRUE/FALSE.</p></li>
<li><p>parameter_class = class of the extracted parameter. Either "Human delay",
"Seroprevalence", "Severity", "Reproduction number", "Mutations",
"Risk factors", or "Other transmission parameters".</p></li>
<li><p>covidence_id = article identifier used by the Imperial team.</p></li>
</ul></div>
    <div class="section level2">
    <h2 id="source">Source<a class="anchor" aria-label="anchor" href="#source"></a></h2>
    <p>Cuomo-Dannenburg G, McCain K, McCabe R, Unwin HJT, Doohan P, Nash RK,
et al. Marburg Virus Disease outbreaks, mathematical models, and disease
parameters: a Systematic Review. medRxiv; 2023. 2023.07.10.23292424.
Available from: https://www.medrxiv.org/content/10.1101/2023.07.10.23292424v1</p>
    </div>

  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Rebecca Nash, Christian Morgenstern, Cosmo Santoni, Sangeeta Bhatia, Richard Sheppard, Joseph Hicks, Gina Cuomo-Dannenburg, Ruth McCabe, Kelly McCain, Joshua Lambert, Anne Cori, Thomas Rawson, Patrick Doohan, Tristan Naidoo, Shazia Ruybal-Pesántez.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

    </footer></div>





  </body></html>

